OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -500,7 +500,7 @@ static bool IsTurn(Mobile mobile, CPos nextCell, Map map)
 				var nextFacing = map.FacingBetween(nextCell, mobile.ToCell, mobile.Facing);
 				var currentFacing = map.FacingBetween(mobile.ToCell, mobile.FromCell, mobile.Facing);
 				var delta = (nextFacing - currentFacing).Angle;
-				return delta != 0 && (delta < 384 || delta > 640);
+				return delta != 0 && !mobile.Info.AlwaysTurn && (delta < 384 || delta > 640);
```

Comment:
We can probably simplify this by putting a `if (mobile.Info.AlwaysTurn) return false;` at the beginning of the method.